### ISSUE TITLE
Python interface to create cgenie input map files?

### ISSUE THREAD

**Issue by oculus99** [derpycode/cgenie.muffin#202]
There is need of create Python/r or some free language program to create input files for
cgenie, because all users does not have Matlab.

**Comment by derpycode** [comment]
Good point, although the configuration files in question are rarely generated by anyone, only if a completely new past continental configuration is needed, or a specific abstracted 'fake' world created.
We have a good set of past/geological configurations spanning the past almost 600 million years, may of which will be published next(!) week.
Model experiments are configured with a simple text file.
But I'll keep your point in mind.

**Comment by oculus99** [comment]
I think that is important issue for worldbuilders and
exoplanet researchers.

I really hope, that at some time you can develop
freely availeble input files generator.





________________________________
Lähettäjä: Andy Ridgwell ***@***.***>
Lähetetty: keskiviikko 10. elokuuta 2022 21.29
Vastaanottaja: derpycode/cgenie.muffin ***@***.***>
Kopio: oculus99 ***@***.***>; Author ***@***.***>
Aihe: Re: [derpycode/cgenie.muffin] Python interface to create cgenie input map files? (Issue #202)


Good point, although the configuration files in question are rarely generated by anyone, only if a completely new past continental configuration is needed, or a specific abstracted 'fake' world created.
We have a good set of past/geological configurations spanning the past almost 600 million years, may of which will be published next(!) week.
Model experiments are configured with a simple text file.
But I'll keep your point in mind.
